dpc-common/src/main/java/gov/cms/dpc/fhir/dropwizard/handlers/BundleHandler.java

```diff
@@ -83,4 +84,20 @@ private Bundle generateBaseBundle(Annotation[] annotations, int entryCount) {
 
         return bundle;
     }
+
+    public static <T extends DomainResource> Bundle convertToBundle(Collection<T> resources) {
+        final Bundle bundle = new Bundle();
+        bundle.setType(Bundle.BundleType.SEARCHSET);
+        resources.forEach(r ->
+                bundle.addEntry(new Bundle.BundleEntryComponent().setResource(r))
+        );
+        return bundle;
+    }
+
+    public static Bundle convertToSummaryBundle(int total) {
+        Bundle bundle = new Bundle();
+        bundle.setType(Bundle.BundleType.SEARCHSET);
+        bundle.setTotal(total);
+        return bundle;
```

Comment:
```suggestion
 return generateBaseBundle(new Annotation[0], total);
```

You'd have to make `generateBaseBundle` static, too, but might as well DRY things up!

You might be able to do something similar with `convertToBundle` and `writeTo`, but it would probably require pulling out some shared code.  Your call if it's worth the effort.

Reply:
Thanks for your feedback! That's a good point! I broke out logic from `writeTo()` into a new method `setBundleEntry()`. Let me know what you think!